### ISSUE TITLE
ObsPluginHelpers.cmake outputs `Depends:` `obs-studio (>= 27.0.0)` and `libqt5*`

### ISSUE THREAD

**Issue by paulpv** [obsproject/obs-plugintemplate#64]
### Operating System Info

Ubuntu 22.04

### OBS Studio Version

29.0.2

### OBS Studio Log URL

N/A

### Expected Behavior

This plugin template is for OBS >=28 QT6 plugins only...right?
So, its generated Linux .deb files should have a `control` file that says something like:
```
...
Depends: obs-studio (>= 28.0.0), libqt6coresomething (>= 6.x.x), libqt6guisomething (>= 6.x.x), libqt6widgetssomething (>= 6.x.x)
...
```

### Current Behavior

The obs-ndi plugin that I work on has a branch based on this `obs-plugintemplate`.
I unzipped the plugin's .deb file generated by this template's build it says:
```
...
Depends: obs-studio (>= 27.0.0), libqt5core5a (>= 5.9.0~beta), libqt5gui5 (>= 5.3.0), libqt5widgets5 (>= 5.7.0)
...
```

It seems to install OK (the jury is still out on that), but I found this very odd/unexpected.

### Steps to Reproduce

1. build a linux x64 plugin
2. unzip the generated .deb file
3. unzip the control.tar.gz file
4. look at the control file

### Anything else we should know?

The above hard coded Depends line is taken straight from these lines of code:
https://github.com/obsproject/obs-plugintemplate/blob/86ae5b4de6454ce42f8ce471054faa34ef66fef0/cmake/ObsPluginHelpers.cmake#L507-L509
```
      set(CPACK_DEBIAN_PACKAGE_DEPENDS
          "obs-studio (>= 27.0.0), libqt5core5a (>= 5.9.0~beta), libqt5gui5 (>= 5.3.0), libqt5widgets5 (>= 5.7.0)"
      )
```

**Comment by paulpv** [comment]
Looks like this would be fixed by https://github.com/obsproject/obs-plugintemplate/pull/47/

**Comment by RytoEX** [comment]
#47 was closed by #76, so this should be resolved.
